### PR TITLE
add workaround for kms driver bug in bullseye

### PR DESCRIPTION
**Pull request by copykatze**
Took me a while to find this info. Since this was the first place I was looking for, maybe it helps others who look here.

### CODE REVIEW

File: documentation/asciidoc/accessories/display/troubleshooting.adoc [hunk 1]
```diff
@@ -1,5 +1,44 @@
 == Troubleshooting the Display
 
+[discrete]
+=== Are you running raspian bullseye and the display turns off during the boot sequence?
```
Comment (lurch) [inline documentation/asciidoc/accessories/display/troubleshooting.adoc:4]: It's now "Raspberry Pi OS", not "Raspbian"

File: documentation/asciidoc/accessories/display/troubleshooting.adoc [hunk 2]
```diff
@@ -1,5 +1,44 @@
...
+Wait for the pi to power down. Remove the power plug and wait a few seconds to power-cycle the display, before pluging it back in to restart.
+
+If the problem still persists, one possible workaround is to disable the kms driver:
+
+[,bash]
+----
+sudo nano /boot/config.txt
+----
+
+Add a "#" to the line "" to disable the kde driver, so the line looks like this:
```
Comment (lurch) [inline documentation/asciidoc/accessories/display/troubleshooting.adoc:26]: kde???

**Comment by lurch** [comment]
ping @pelwell

**Comment by JamesH65** [comment]
I'm not convinced this is an appropriate change, should the docs contain workaround for bugs that are soon to be fixed?

**Comment by aallan** [comment]
> I'm not convinced this is an appropriate change, should the docs contain workaround for bugs that are soon to be fixed?

Yeah, I'm also really unsure about that. I'm pondering. I'll keep an eye on raspberrypi/linux#4686 for a day or two and see what I think.

**Comment by copykatze** [comment]
> I'm not convinced this is an appropriate change, should the docs contain workaround for bugs that are soon to be fixed?

Do as you feel right, I am not invested. To me it seemed like the right thing to suggest after looking for a solution at this place first and then spending some time in finding a fix. It seems there are other people wondering about this in some forums.

I agree that a temporary workaround with a fix in sight does not belong here. But as they say over at https://github.com/raspberrypi/linux/issues/4686: "There are a number of issues that have been noted since the Bullseye release. Triaging and then fixing them all takes a little time" - which to me sounds like it could take a while...

File: documentation/asciidoc/accessories/display/troubleshooting.adoc [hunk 3]
```diff
@@ -1,5 +1,44 @@
...
+First, make sure the system is up-to-date and power-cycle the display to see if the update fixed the problem:
+
+[,bash]
+----
+sudo apt update 
+sudo apt upgrade
+sudo shutdown -h now
+----
+
+Wait for the pi to power down. Remove the power plug and wait a few seconds to power-cycle the display, before pluging it back in to restart.
```
Comment (lurch) [inline documentation/asciidoc/accessories/display/troubleshooting.adoc:17]: pluging -> plugging
(same typo also appears again later)

File: documentation/asciidoc/accessories/display/troubleshooting.adoc [hunk 4]
```diff
@@ -1,5 +1,44 @@
 == Troubleshooting the Display
 
+[discrete]
+=== Are you running Raspberry Pi OS bullseye and the display turns off during the boot sequence?
+
+There is a bug that requires a workaround to get the display working until the bug is fixed: https://github.com/raspberrypi/linux/issues/4686
+
+First, make sure the system is up-to-date and power-cycle the display to see if the update fixed the problem:
+
+[,bash]
```
Comment (aallan) [inline documentation/asciidoc/accessories/display/troubleshooting.adoc:10]: If it does get merged — which I'm still pondering — this bit will get shortened down considerably.

**Comment by copykatze** [comment]
Sorry, I did not mean to cause more trouble that this information might be worth. I withdraw the suggestion. The info about the current problem and workaround is out there and can be found.

**Pull request closed without merging**